champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/SkillsCommand.java

```diff
@@ -7,17 +7,25 @@
 import me.mykindos.betterpvp.champions.champions.builds.RoleBuild;
 import me.mykindos.betterpvp.champions.champions.roles.RoleManager;
 import me.mykindos.betterpvp.core.client.Client;
+import me.mykindos.betterpvp.core.client.Rank;
 import me.mykindos.betterpvp.core.command.Command;
 import me.mykindos.betterpvp.core.components.champions.Role;
+import me.mykindos.betterpvp.core.config.Config;
 import me.mykindos.betterpvp.core.utilities.UtilMessage;
 import net.kyori.adventure.text.Component;
 import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 
+import java.util.List;
 import java.util.Optional;
 
 @Singleton
 public class SkillsCommand extends Command {
+    @Inject
+    @Config(path = "command/skills/seeOtherPlayerSkillsRank", defaultValue = "TRIAL_MOD" )
```

Comment:
path should be delimited with . instead of /

Reply:
I have no idea why I did it that way